### ISSUE TITLE
Change Copyright header

### ISSUE THREAD

**Issue by mkurz** [playframework/play-meta#211]
Right now:
* `Copyright (C) Lightbend Inc. <https://www.lightbend.com>`

Suggestion for new one:
* `2011-2021 Lightbend Inc., since 2022 the original author or authors from the Play Framework project`

Actually @ihostage came up with the JHipster one in slack. So that is what JHipster uses:
https://github.com/jhipster/generator-jhipster/blob/9f8da38a6251416c9b700f35b9533b157d537b4d/cli/commands.js#L2-L5
```java
 * Copyright 2013-2022 the original author or authors from the JHipster project.
 *
 * This file is part of the JHipster project, see https://www.jhipster.tech/
 * for more information.
 *
```

BTW: This is for source code, on the website they just use:
<kbd><img width="500" src="https://user-images.githubusercontent.com/644927/150431526-6b9e9613-2d53-4546-b02d-e9adda8de133.png" /></kbd>

**Comment by octonato** [comment]
> 2011-2021 Lightbend Inc., since 2022 the original author or authors from the Play Framework project

I find this confusing. 
I guess the intention is to say, starting from 2022 we consider Lightbend the "original author". 

But that's not quite true. The original author is Guillaume. 

I think we can just have 

```
Copyright 2011-2021 (C) Lightbend Inc. <https://www.lightbend.com>
```

And then add an extra header starting from 2022.

**Comment by mkurz** [comment]
Repo to update:
:question:

- [ ] [playframework](https://github.com/playframework/playframework)
- [ ] [playframework.com](https://github.com/playframework/playframework.com)
- [ ] [interplay](https://github.com/playframework/interplay)


- [ ] [play-doc](https://github.com/playframework/play-doc)
- [ ] [play-ebean](https://github.com/playframework/play-ebean)
- [ ] [play-file-watch](https://github.com/playframework/play-file-watch)
- [ ] [play-grpc](https://github.com/playframework/play-grpc)
- [ ] [play-json](https://github.com/playframework/play-json)
- [ ] [play-mailer](https://github.com/playframework/play-mailer)
- [ ] [play-samples](https://github.com/playframework/play-samples)
- [ ] [play-slick](https://github.com/playframework/play-slick)
- [ ] [play-soap](https://github.com/playframework/play-soap)
- [ ] [play-socket.io](https://github.com/playframework/play-socket.io)
- [ ] [play-webgoat](https://github.com/playframework/play-webgoat)
- [ ] [play-ws](https://github.com/playframework/play-ws)


- [ ] [anorm](https://github.com/playframework/anorm)
- [ ] [cachecontrol](https://github.com/playframework/cachecontrol)
- [ ] [netty-reactive-streams](https://github.com/playframework/netty-reactive-streams)
- [ ] [omnidoc](https://github.com/playframework/omnidoc)
- [ ] [scalatestplus-play](https://github.com/playframework/scalatestplus-play)
- [ ] [twirl](https://github.com/playframework/twirl)


- [ ] [play-java-seed.g8](https://github.com/playframework/play-java-seed.g8)
- [ ] [play-scala-seed.g8](https://github.com/playframework/play-scala-seed.g8)
- [ ] [play-java-react-seed](https://github.com/playframework/play-java-react-seed)
- [ ] [play-scala-react-seed](https://github.com/playframework/play-scala-react-seed)
- [ ] [play-java-angular-seed](https://github.com/playframework/play-java-angular-seed)
- [ ] [play-scala-angular-seed](https://github.com/playframework/play-scala-angular-seed)


- [ ] [.github](https://github.com/playframework/.github)
- [ ] [play-meta](https://github.com/playframework/play-meta)
- [ ] [play-generated-docs](https://github.com/playframework/play-generated-docs)

**Comment by ihostage** [comment]
> To finally finish this I suggest:
> 
> ```
> Copyright from 2022 (C) Contributors to the Play Framework project <https://github.com/playframework>
> Copyright 2011-2021 (C) Lightbend Inc. <https://www.lightbend.com>
> ```
> 
> Taken from https://www.linuxfoundation.org/blog/blog/copyright-notices-in-open-source-software-projects

I agree. Although **"Copyright The Play Framework Contributors."** looks shortly and is liked me more 😄

**Comment by mkurz** [comment]
>  liked me more

:wink: I don't care. Actually I am fixing sbt-header right now because it is not working correctly.

**Comment by mkurz** [comment]
OK, I I was thinking again about this. Let's just do for the `main` branch
```
Copyright (C) The Play Framework Contributors <https://github.com/playframework>
```
So Lightbend will be dropped (they are contributors also so IMHO they are included in this phrase anyway). No year and the url at the end (using github since that were all other repos are and I want to use phrase for all repos)

**Comment by mkurz** [comment]
The old copyright header is gone from every repo's `main` branch (I just checked). Repos that did/do have copyright headers use the new one now.
IMHO we can close this issue. We still can set up sbt-header in some repos if we find it useful (for some repos it is not necessary IMHO, e.g. play-samples or the see projects)

@ihostage if you think we should keep it open please let me know.

**Comment by ihostage** [comment]
I use this issue only to add sbt-header plugin into all repos. But I agree with you that subject "Change Copyright header" is done ✅ 
I think we can create a separate issue with description as [this comment](https://github.com/playframework/play-meta/issues/211#issuecomment-1281501019) and close this.
